department-of-veterans-affairs/caseflow
# Resolve bugs for convert-to-virtual flow on daily docket page

Resolves #14797 

### Description
Resolves bugs with daily docket page discovered during dogfooding session

### Acceptance Criteria
- [ ] All bugs from dogfooding for daily docket are resolved

### Testing Plan
1. Shadow user BVASYELLOW
1. Navigate to the hearing schedule and select a Central hearing: http://localhost:3000/hearings/schedule
1. Click the 'Edit Hearing Details' link
1. Change the type to Virtual
1. Go back to the daily docket page for this hearing
1. Go through the list of items in the ticket and ensure all of the issues are resolved
